parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -266,16 +266,24 @@ pullContainerImage() {
 cleanUpContainerImages() {
     # TODO remove all unused container images at runtime
     docker rmi $(docker images --format '{{.Repository}}:{{.Tag}}' | grep -v "${KUBERNETES_VERSION}$" | grep 'hyperkube') &
+    pids="$pids $!"
     docker rmi $(docker images --format '{{.Repository}}:{{.Tag}}' | grep -v "${KUBERNETES_VERSION}$" | grep 'cloud-controller-manager') &
+    pids="$pids $!"
     if [ "$IS_HOSTED_MASTER" = "false" ]; then
         echo "Cleaning up AKS container images, not an AKS cluster"
         docker rmi $(docker images --format '{{.Repository}}:{{.Tag}}' | grep 'hcp-tunnel-front') &
+        pids="$pids $!"
         docker rmi $(docker images --format '{{.Repository}}:{{.Tag}}' | grep 'kube-svc-redirect') &
+        pids="$pids $!"
         docker rmi $(docker images --format '{{.Repository}}:{{.Tag}}' | grep 'nginx') &
+        pids="$pids $!"
     fi
 
     # TODO: remove once ACR is available on Azure Stack
     docker rmi registry:2 &
+    pids="$pids $!"
+
+    wait $pids
```

Comment:
These changes don't specifically relate to disabling unattended upgrades, right? This change would block on returning control from `cleanUpContainerImages` until all the docker images are deleted, right?

Reply:
Yeah I've just realized that.

It relates to the reboot of the node at the end of the cse if security upgrades have been installed.

I'll move it out and make a separate PR.

Reply:
thanks!